extensions/producthunt/src/components/ProductDetailView.tsx

```diff
@@ -118,7 +119,7 @@ export function ProductDetailView({
                     });
                     open(product.hunter.profileUrl);
                   } else if (product.hunter?.username) {
-                    const profileUrl = `https://www.producthunt.com/@${product.hunter.username}`;
+                    const profileUrl = `${HOST_URL}/@${product.hunter.username}`;
```

Comment:
```suggestion
                    const profileUrl = `${HOST_URL}@${product.hunter.username}`;
```
As you already have the slash in the constant, you could remove it here ;)

Reply:
Ugg, good catch!